### DISCUSSION TITLE
Codewars not showing topics below difficulity

### DISCUSSION THREAD

**Discussion by RazManva** [codewars/codewars.com#2541] [General]
Previously Codewars was showing different topics to select below difficulty. 
Topics like fundamental, loops string were displayed below difficulty 8 kyu to 1 kyu.   
Now suddenly it has stopped showing and it is very difficulty to search katas topicwise.

Requesting you to support.

**Comment by kazk** [comment]
This was affected by #2534, and I just deployed a fix.

**Reply by RazManva** [reply]
Thanks a lot.

Resolved totally.
